assembl/static2/js/app/components/debate/brightMirror/fictionCommentForm.jsx

```diff
@@ -28,12 +32,14 @@ const COMMENT_SUBMIT_BUTTON_ID = 'comment-submit-button';
 
 class FictionCommentForm extends Component<FictionCommentFormProps, FictionCommentFormState> {
   static defaultProps = {
+    commentValue: '',
+    editMode: false,
     rowsForTextarea: 2
   };
 
   state = {
-    commentTextareaValue: '',
-    showFormActionButtons: false
+    commentTextareaValue: this.props.commentValue ? this.props.commentValue : EMPTY_STRING,
```

Comment:
destructuring props

Reply:
@amensouissi that would force me to use 'constructor' method in order to destructure the props. Do you have an other syntax in mind ?